crates/chain-config/src/config/consensus.rs

```diff
@@ -1,10 +1,33 @@
-use fuel_core_types::fuel_types::Address;
+use std::time::Duration;
+
+use fuel_core_types::{
+    fuel_tx::Input,
+    fuel_types::Address,
+};
 use serde::{
     Deserialize,
     Serialize,
 };
 
+use crate::default_consensus_dev_key;
+
 #[derive(Clone, Debug, Deserialize, Serialize, Eq, PartialEq)]
 pub enum ConsensusConfig {
-    PoA { signing_key: Address },
+    PoA {
```

Comment:
Because the network participants can't verify effects from other fields than `signing_key`, it can't be part of the `ChainConfig::ConsensusConfig`. 

It should be part of the `fuel_core_poa::Config`